src/vs/workbench/contrib/chat/browser/actions/chatTitleActions.ts

```diff
@@ -56,12 +56,12 @@ export function registerChatTitleActions() {
 					id: MenuId.ChatMessageFooter,
 					group: 'navigation',
 					order: 1,
-					when: ContextKeyExpr.and(ChatContextKeys.isResponse, ChatContextKeys.responseHasError.negate())
+					when: ContextKeyExpr.and(ChatContextKeys.isResponse, ChatContextKeys.responseHasError.negate(), ContextKeyExpr.has('telemetry.disableFeedback'))
```

Comment:
Should it be `config.telemetry.disableFeedback` or are you writing this as a context key somewhere?

Reply:
And should it be negated?

Reply:
oh yes was unaware that's what we had to do for settings. also you're right - for some reason i had it switched when i tested it 🤕